dot/network/gossip_test.go

```diff
@@ -84,10 +84,12 @@ func TestGossip(t *testing.T) {
 		Digest: types.NewDigest(),
 	}
 
-	_, err = nodeA.host.send(addrInfoB.ID, "", announceMessage)
+	time.Sleep(10 * time.Second)
+
+	_, err = nodeA.host.send(addrInfoB.ID, "/gossamer/test/0/block-announces/1", announceMessage)
 	require.NoError(t, err)
 
-	time.Sleep(TestMessageTimeout)
+	time.Sleep(10 * time.Second)
```

Comment:
locally the test passes even with one second delay. I am guessing the delay requirement might be slightly different. 10s seems little high though. 
But seems like only potential way to find an optimum delay would be to run ci bunch of times.